File: src/test/java/com/jigumulmi/admin/banner/BannerPlaceMappingRepository.java
package com.jigumulmi.admin.banner;

import com.jigumulmi.banner.domain.BannerPlaceMapping;
import java.util.List;
import org.springframework.data.jpa.repository.JpaRepository;

public interface BannerPlaceMappingRepository extends JpaRepository<BannerPlaceMapping, Long> {

    List<BannerPlaceMapping> findAllByBannerId(Long bannerId);
    List<BannerPlaceMapping> findAllByBannerIdIn(List<Long> bannerIdList);
}

File: admin/src/test/kotlin/com/jigumulmi/banner/BannerPlaceMappingRepository.kt
package com.jigumulmi.banner

import com.jigumulmi.banner.domain.BannerPlaceMapping
import org.springframework.data.jpa.repository.JpaRepository

interface BannerPlaceMappingRepository : JpaRepository<BannerPlaceMapping?, Long?> {
    fun findAllByBannerId(bannerId: Long?): List<BannerPlaceMapping?>?
}
